### PR TITLE
Feature/chart with service

### PR DESCRIPTION
**Pull request by khushbu-maurya**
added static data for notification for testing propose

### CODE REVIEW

File: src/main/webapp/app/home/notification-set.component.html [hunk 1]
```diff
@@ -1,4 +1,74 @@
...
+                        <svg height='200' width='280' id="StackedHorizontal"></svg>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+<div class="text-center" ng-show="vm.count > 0">
```
Comment (johibkhan2) [inline src/main/webapp/app/home/notification-set.component.html]: seems line got removed .pls replace the vm.count>0 to below line
ng-show="vm.noOfPages > 1"

File: src/main/webapp/app/home/notification-set.component.js [hunk 2]
```diff
@@ -20,23 +20,214 @@
...
 
         active();
 
-        function active(){
+        function active() {
             // vm.alerts=vm.releaseAlert.alerts;
             // vm.count=vm.releaseAlert.count;
             connectWebSocket();
             getScheduledReportsCount();
+            vm.reports=[
```
Comment (johibkhan2) [inline src/main/webapp/app/home/notification-set.component.js]: are you testing with hard code content

File: src/main/webapp/app/home/notification-set.component.html (hunk 1 above)
Comment (khushbu-maurya) [reply src/main/webapp/app/home/notification-set.component.html]: okay

File: src/main/webapp/app/home/notification-set.component.js (hunk 2 above)
Comment (khushbu-maurya) [reply src/main/webapp/app/home/notification-set.component.js]: yes I have already added in description 
but now I am working with API data

**Pull request merged**